src/components/metadata/templates/LocalContextsField.js

```diff
@@ -0,0 +1,241 @@
+    const fieldError = getFormError(field.name, touched, errors);
+    const projectID = parseProjectID(projectHubURI);
+
+    const { data: project, isFetching } = useQuery({
+        queryKey: [LOCAL_CONTEXTS_QUERY_KEY, projectID],
+        queryFn: () =>
+            getLocalContextsProject({
+                projectID,
+            }),
+        enabled: projectHubURI && !fieldError,
+        onSuccess: (project) => {
+            let newValue = avu.value || "";
+
+            const projectLabels = [
+                ...(project?.notice?.filter((label) => label?.name) || []),
+                ...(project?.bc_labels?.filter((label) => label?.name) || []),
+                ...(project?.tk_labels?.filter((label) => label?.name) || []),
+            ];
+
+            if (projectLabels.length === 1) {
+                newValue =
+                    projectLabels[0].label_text ||
+                    projectLabels[0].default_text;
+            } else {
+                newValue = t("localContextsAttrDefaultValue", {
+                    projectTitle: project?.title,
+                    projectPage: project?.project_page,
+                });
+            }
+
+            const newLabels = projectLabels.map((label) => label.name);
+
+            let newAVUs = avu.avus || [];
+
+            const currentLabels = newAVUs
+                .filter(
+                    (childAVU) => childAVU.attr === LocalContextsAttrs.RIGHTS_ID
+                )
+                ?.map((childAVU) => childAVU.value);
+
+            const missingLabels = newLabels.filter(
+                (label) => !currentLabels.includes(label)
+            );
+
+            const extraLabels = currentLabels.filter(
+                (label) => !newLabels.includes(label)
+            );
+
+            const requiresUpdate =
+                avu.value !== newValue ||
+                missingLabels.length > 0 ||
+                extraLabels.length > 0;
+
+            if (requiresUpdate) {
+                newAVUs = newAVUs.filter(
+                    (childAVU) =>
+                        childAVU.attr !== LocalContextsAttrs.RIGHTS_URI &&
+                        childAVU.attr !== LocalContextsAttrs.RIGHTS_ID_SCHEME &&
+                        childAVU.attr !== LocalContextsAttrs.SCHEME_URI &&
+                        (childAVU.attr !== LocalContextsAttrs.RIGHTS_ID ||
+                            newLabels.includes(childAVU.value))
+                );
+
+                newAVUs = [
+                    ...newAVUs,
+                    rightsURIAVU,
+                    schemeURIAVU,
+                    rightsIDSchemeURIAVU,
+                    ...missingLabels.map((label) => ({
+                        attr: LocalContextsAttrs.RIGHTS_ID,
+                        value: label,
+                        unit: "",
+                    })),
+                ];
+
+                onUpdate({ ...avu, value: newValue, avus: newAVUs });
+            }
+        },
```

Comment:
This logic in the Local Contexts Hub project query's `onSuccess` function synchronizes the labels defined in the project with the DE metadata. It also ensures the other child AVUs required by DataCite v4.2 are included.

The top-level AVU's value is based on suggestions from [DataCite's docs](https://support.datacite.org/docs/local-contexts-notices-and-labels).

The required AVU and child AVUs should be like the following:
```json
{
      "attr": "LocalContexts",
      "value": "The \"January Test\" project has Labels and/or Notices applied through the Local Contexts Hub. For more information, refer to the project page: https://sandbox.localcontextshub.org/projects/adcbca87-081a-4e2c-b6af-bff12fe3b7b0",
      "unit": "",
      "avus": [
        {
          "attr": "rightsIdentifier",
          "value": "Traditional Knowledge (TK) Notice",
          "unit": ""
        },
        {
          "attr": "rightsIdentifier",
          "value": "Biocultural (BC) Notice",
          "unit": ""
        },
        {
          "attr": "rightsIdentifier",
          "value": "Attribution Incomplete Notice",
          "unit": ""
        },
        {
          "attr": "rightsURI",
          "value": "https://sandbox.localcontextshub.org/projects/adcbca87-081a-4e2c-b6af-bff12fe3b7b0/",
          "unit": ""
        },
        {
          "attr": "schemeURI",
          "value": "https://localcontexts.org",
          "unit": ""
        },
        {
          "attr": "rightsIdentifierScheme",
          "value": "Local Contexts",
          "unit": ""
        }
      ]
    }
```